UnitsNet/GeneratedCode/Quantities/Angle.g.cs

```diff
@@ -786,6 +788,15 @@ public bool Equals(Angle other)
 
         #pragma warning restore CS0809
 
+        /// <summary>
+        ///     Returns the hash code for this instance.
+        /// </summary>
+        /// <returns>A hash code for the current Angle.</returns>
+        public override int GetHashCode()
+        {
+            return new { Info.Name, Value, Unit }.GetHashCode();
```

Comment:
Except for netstandard, this can be `return HashCode.Combine(Info.Name, Value, Unit)` and probably save an allocation.

Reply:
fixed in https://github.com/angularsen/UnitsNet/pull/1587/commits/2ea2f0bd57c30b7baeef31a5b35cce6569d14efc (u